lib/ex_prosemirror/html/form.ex

```diff
@@ -10,8 +10,9 @@ defmodule ExProsemirror.HTML.Form do
 
   ## Options
 
-  - marks: mark tags to use defined in your ProseMirror config
-  - blocks: block tags to use defined in your ProseMirror config
+  - `marks`: mark tags to use defined in your ProseMirror config
+  - `blocks`: block tags to use defined in your ProseMirror config
+  - `type`: Type of input to use (see: `ExProsemirror.Config` for more informations)
```

Comment:
```suggestion
  - `type`: type of input to use (see: `ExProsemirror.Config` for more informations)
```

All the other ones do not begin with a capital, so I would make them all capital or none